docs/bundles/plain.md

```diff
@@ -29,16 +58,123 @@ manifests
 └── deployment.yaml
 ```
 
+For a bundle `git` repo, any directory that contains only static Kubernetes manifests checked into a `git` repository
```

Comment:
I don't think this is quite true. A plain bundle must have a `manifests` directory (yes, specifically named `manifests`) at its root. So it can't be _any_ directory. The bundle root directory has to contain a `manifests` directory.

Reply:
TBH, I don't think I'd talk about the source types here though. The bundle format and the sourcing mechanism are completely decoupled in the rukpak design.

Reply:
Removed it in https://github.com/operator-framework/rukpak/pull/642/commits/56cfbae769c96755690befe3628c45b9e36ee244